Task_7/DeliveryTrucks/truck_delivery_rayus007/db_connector_redis.py

```diff
@@ -0,0 +1,23 @@
+import redis
+from truck_delivery_rayus007.db_connector import DBConnector
+
+
+class DbConnectorRedis(DBConnector):
+    def __init__(self):
+        super(DbConnectorRedis, self).__init__()
+
+    def get_connection(self):
+        if self.connection:
+            return self.connection
+        else:
+            return redis.Redis(host='localhost', port=6379, db=0)
+
+
+def test_connection():
```

Comment:
nice, this was for testing the connection right ?

Reply:
Yes, I am using it much better in the next task.